edx_solutions_api_integration/test_utils.py

```diff
@@ -275,19 +272,13 @@ def connect_signals():
         """
         connects signals defined in solutions apps
         """
-        SignalHandler.course_published.connect(
-            listener_in_course_metadata, dispatch_uid='course_metadata'
-        )
```

Comment:
Since course metatdata is no longer being used, we no longer need to keep it up to date.